packages/apps/graph-client/src/constants/search.ts

```diff
@@ -0,0 +1,38 @@
+export enum SearchType {
+  GasEstimation = 'gasEstimation',
```

Comment:
Shall we make this `gas-estimation` for consistency?

Reply:
Fixed